test/core/is-same.test.ts

```diff
@@ -0,0 +1,21 @@
+import * as depGraphLib from '../../src';
+import * as helpers from '../helpers';
+
+describe('isSame', () => {
+  const simpleGraphJson = helpers.loadFixture('simple-graph.json');
+  const goofGraphJson = helpers.loadFixture('goof-graph.json');
+
+  const simpleGraphA = depGraphLib.createFromJSON(simpleGraphJson);
+  const simpleGraphB = depGraphLib.createFromJSON(simpleGraphJson);
+  const goofGraph = depGraphLib.createFromJSON(goofGraphJson);
+
+  test('compare same graphs', async () => {
+    expect(simpleGraphA.isSame(simpleGraphB)).toBe(true);
+    expect(simpleGraphB.isSame(simpleGraphA)).toBe(true);
+  });
+
+  test('compare different graphs', async () => {
```

Comment:
please add another test where the graphs are only slightly different - in example same packages but different relations

Reply:
done